core/src/test/java/org/elasticsearch/index/query/MissingQueryBuilderTest.java

```diff
@@ -68,96 +63,8 @@ private String getRandomFieldName() {
     }
 
     @Override
-    protected Query doCreateExpectedQuery(MissingQueryBuilder queryBuilder, QueryParseContext context) throws IOException {
-        final boolean existence = queryBuilder.existence();
-        final boolean nullValue = queryBuilder.nullValue();
-        String fieldPattern = queryBuilder.fieldPattern();
+    protected void doAssertLuceneQuery(MissingQueryBuilder queryBuilder, Query query, QueryParseContext context) throws IOException {
```

Comment:
Not sure if given the complexity of the query creation here it is possible to do something similar as in ExistsQueryBuilderTest here for example?